dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -140,30 +342,163 @@ export default defineComponent({
     };
   },
   name: "CreateEUTaxonomyForFinancials",
-  components: { DataPointFormElement, FailedUpload, FormKit, SuccessUpload, Card },
+  components: {
+    JumpLinksSection,
+    FailedUpload,
+    FormKit,
+    SuccessUpload,
+    UploadFormHeader,
+    Card,
+
+    UploadReports,
+    BasicInformationFields,
+
+    PrimeButton,
+    Calendar,
+    MultiSelect,
+    KPIfieldSet,
+  },
 
-  data: () => ({
-    innerClass: {
-      "formkit-inner": false,
-      "p-inputwrapper": true,
+  data() {
+    return {
+      formInputsModel: {} as CompanyAssociatedDataEuTaxonomyDataForFinancials,
+      filesToUpload: [] as ExtendedFile[],
+      uploadFiles: [] as ExtendedCompanyReport[],
+      fiscalYearEnd: "" as Date | "",
+      convertedFiscalYearEnd: "",
+      reportingPeriod: new Date(),
+      assuranceData: [
+        { label: "None", value: "None" },
+        { label: "LimitedAssurance", value: "LimitedAssurance" },
+        { label: "ReasonableAssurance", value: "ReasonableAssurance" },
+      ],
+      maxFileSize: UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+      euTaxonomyKPIsModel: euTaxonomyPseudoModelAndMappings,
+      euTaxonomyKpiNameMappings,
+      euTaxonomyKpiInfoMappings,
+      checkCustomInputs,
+      formatBytesUserFriendly,
+      route: useRoute(),
+      waitingForData: false,
+      editMode: false,
+
+      postEuTaxonomyDataForFinancialsProcessed: false,
+      messageCount: 0,
+      postEuTaxonomyDataForFinancialsResponse: null as AxiosResponse<DataMetaInformation> | null,
+      humanizeString: humanizeString,
+      onThisPageLinksStart: [
+        { label: "Upload company reports", value: "uploadReports" },
+        { label: "Basic information", value: "basicInformation" },
+        { label: "Assurance", value: "assurance" },
+        { label: "Add KPIs", value: "addKpis" },
+      ],
+      onThisPageLinks: [] as { label: string; value: string }[],
+      kpisModel: [
+        { label: "Credit Institution", value: "creditInstitutionKpis" },
+        { label: "Investment Firm", value: "investmentFirmKpis" },
+        { label: "Insurance & Re-insurance", value: "insuranceKpis" },
+        { label: "Asset Management", value: "assetManagementKpis" },
```

Comment:
Just something I noticed: This seems a little redundant to me since you already have this mapping in `euTaxonomyPseudoModelAndMappings`.
Would it be possible to use that instead of hardcoding the info here as strings?

Reply:
There is no such specific one.